### PR TITLE
Optimize by add woff2, and extra styles

### LINKED ISSUE #17: What happened to extra styles?
Things like `ph-lg`, `ph-fw`, etc are all missing :(

### PR DESCRIPTION
**Pull request by ThatOneCalculator**
Pretty much all modern browsers support the `woff2` format, which is ~33% the size of the `woff` counterparts.

~~Also closes #17 by adding back the 1.x styles.~~

### CODE REVIEW

**Comment by rektdeckard** [comment]
Thanks for the PR. Happy to accept the WOFF2 format, but we aren't interested in shipping utility classes any more, for reasons I mentioned in #17

**Comment by ThatOneCalculator** [comment]
Done

**Comment by ThatOneCalculator** [comment]
All extra cleanup should be done

**Comment by rektdeckard** [comment]
You'll also need to bump the CDN version in https://github.com/ThatOneCalculator/phosphor-icons/blob/f7fa23ae2e2ed0847a397c36c35ab97cdd70bb3a/src/index.js#L8

**Comment by ThatOneCalculator** [comment]
Hopefully that'll be my last time saying "done" :smile:

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+.vscode/settings.json
```
Comment (rektdeckard) [inline .gitignore:2]: I think you can ignore the whole folder here.

Comment (rektdeckard) [reply .gitignore:2]: Even though I definitely have this committed in other phosphor repos :sweat_smile:

**Comment by rektdeckard** [comment]
Looks great. Thanks for the addition!

**Comment by ThatOneCalculator** [comment]
:rocket:

**Comment by ThatOneCalculator** [comment]
Thank you so much! Will an update to npm be pushed soon, so I can upgrade my project?

**Pull request merged**